scripts/make_bulk_data.sh

```diff
@@ -84,15 +84,15 @@ fjcintegrateddatabase_csv_filename="fjc-integrated-database-$(date -I).csv"
 
 # search_opinioncluster
 opinioncluster_fields='(
-	       id, date_created, date_modified, judges, date_filed,
-	       date_filed_is_approximate, slug, case_name_short, case_name,
-	       case_name_full, scdb_id, scdb_decision_direction, scdb_votes_majority,
-	       scdb_votes_minority, source, procedural_history, attorneys,
-	       nature_of_suit, posture, syllabus, headnotes, summary, disposition,
-	       history, other_dates, cross_reference, correction, citation_count,
-	       precedential_status, date_blocked, blocked, filepath_json_harvard, docket_id,
-	       arguments, headmatter
-	   )'
+       id, date_created, date_modified, judges, date_filed,
+       date_filed_is_approximate, slug, case_name_short, case_name,
+       case_name_full, scdb_id, scdb_decision_direction, scdb_votes_majority,
+       scdb_votes_minority, source, procedural_history, attorneys,
+       nature_of_suit, posture, syllabus, headnotes, summary, disposition,
+       history, other_dates, cross_reference, correction, citation_count,
+       precedential_status, date_blocked, blocked, filepath_json_harvard, filepath_pdf_harvard, docket_id,
+       arguments, headmatter
```

Comment:
I don't usually care too much, but it stuck out at me: Can we wrap at about 80 chars for this, please?